templates/config.edn

```diff
@@ -343,7 +343,12 @@
  ;;     ;use Percent-encoding for other invalid characters
  :file/name-format :triple-lowbar
 
- ;; specify the format of the filename for journal files
+ ;; Specify the format of the filename for journal files.
+ ;; Warning:
+ ;;   Changing this setting may cause issues if not configured correctly.
+ ;;   This configuration option is not retroactive and will only affect newly created journals.
+ ;; You can use any date format with valid characters for your filesystem.
+ ;; The default value is "yyyy-MM-dd".
```

Comment:
Now that the default value is yyyy-MM-dd and the current value is yyyy_MM_dd, is it better to provide conversion tools like filename format?

Reply:
@hezhizhen It's easy to do with bash or even with VS code + regex. This option is for power users and the average user should leave it as is.

Reply:
> @hezhizhen It's easy to do with bash or even with VS code + regex. This option is for power users and the average user should leave it as is.

okay